Clarify wording in BR tool doc (#3810)
pingcap/docs: br/backup-and-restore-tool.md

LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/backupdata`
```

> In the above example, the incremental backup data records the data updates and the DDLs between `(LAST_BACKUP_TS, current PD timestamp]`. When restoring data, BR restores DDLs first and then restores the data.

Reviewer: ```suggestion
In the above example, for the incremental backup data, BR records the data changes and the DDL operations during `(LAST_BACKUP_TS, current PD timestamp]`. When restoring data, BR first restores DDL operations and then the data.
```
